src/_static/student-dashboard.html

```diff
@@ -0,0 +1,315 @@
+<link rel="import" href="/static/polymer-1.2.0/polymer/polymer.html">
+<link rel="import" href="/static/polymer-1.2.0/iron-ajax/iron-ajax.html"/>
+<link rel="import" href="/static/polymer-1.2.0/paper-spinner/paper-spinner.html"/>
+<link rel="import" href="/modules/coursebuilder-student-dashboard/_static/components/utility/unsafe-html.html">
+<link rel="import" href="/modules/coursebuilder-student-dashboard/_static/components/utility/floating-sidebar.html">
+<link rel="import" href="/modules/coursebuilder-student-dashboard/_static/components/course-cards/course-card-grouper.html">
+<link rel="import" href="/modules/coursebuilder-student-dashboard/_static/components/filter-sidebar/filter-sidebar.html">
+<link rel="import" href="/modules/coursebuilder-student-dashboard/_static/dashboard-wrapper.html">
+<link rel="import" href="/modules/coursebuilder-student-dashboard/_static/components/top-bar/top-bar.html">
+
+
+<dom-module id="student-dashboard">
+  <template>
+    <top-bar current-user="[[currentUser]]"></top-bar>
+   
+    <template is="dom-if" if="[[loadDashboard]]">
+      <iron-ajax
+        auto url="/modules/gql/query"
+        params="[[_getDetailedCoursesQuery()]]"
+        handle-as="json"
+        json-prefix=")]}'"
+        on-response="_onDetailedCoursesResponse"
+      ></iron-ajax>
+    </template>
+
+    <!-- The wrapper that embeds the student dashboard -->
+    <dashboard-wrapper loaded="[[dashboardLoaded]]" current-user="[[currentUser]]" dashboardcourses="[[dashboardCourses]]">
+    </dashboard-wrapper>
+    
+    <div id="explorer" class="page-wrapper">
+      <style>
+        #top-content {
+          background: white;
+          border-bottom: 1px solid rgb(229, 229, 229);
+        }
+
+        paper-spinner {
+          position: absolute;
+          left: 32px;
+          top: 32px;
+        }
+      </style>
+
+      <iron-ajax
+        auto url="/modules/gql/query"
```

Comment:
@andela-iakande Should we have this in the module?

Reply:
Yes, because the student dashboard is dependent on the courses. We make ajax calls to the courses which then loads the student dashboard template.